### PR TITLE
updates codebase

### PR DESCRIPTION
**Pull request by jonkiky**

### CODE REVIEW

**Review by skiran86 — APPROVED** [review]

File: src/pages/dashboard/dashboardState.js [hunk 1]
```diff
@@ -0,0 +1,344 @@
+import {
```
Comment (AjayDoddapaneni) [inline src/pages/dashboard/dashboardState.js:1]: Do we need this?

**Review by iksheth — APPROVED** [review]

File: src/pages/dashboard/dashboardState.js (hunk 1 above)
Comment (jonkiky) [reply src/pages/dashboard/dashboardState.js:1]: > Do we need this?

Please check JS pageSpecificStatsController.js:

Comment (jonkiky) [reply src/pages/dashboard/dashboardState.js:1]: pageSpecificStatsController.js:

 import { getFilteredStat, fetchDataForDashboardDataTable } from '../../pages/dashboard/dashboardState';

Comment (AjayDoddapaneni) [reply src/pages/dashboard/dashboardState.js:1]: Can you update this to use dashboardreducer.js  so we don't use this entire file

**Pull request merged**